io.github.kriptolix.Poliedros.metainfo.xml.in

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.kriptolix.Poliedros</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <name>Poliedros</name>
+  <summary>Poliedros is a multi-type dice roller.</summary>
+  <description>
+    <p>Poliedros is a multi-type dice roller. Basic mode allows you to make quick rolls using just the mouse, while Command mode allows for more complex rolls and combinations.</p>
+  </description>
+
+  <developer id="tld.vendor">
+    <name>Diego C Sampaio</name>
+  </developer>
+
+  <!-- Requered: Should be a link to the upstream homepage for the component -->
+  <url type="homepage">https://github.com/kriptolix/Poliedros</url>
+  <!-- Recommended: It is highly recommended for open-source projects to display the source code repository -->
+  <url type="vcs-browser">https://github.com/kriptolix/Poliedros</url>
+  <!-- Should point to the software's bug tracking system, for users to report new bugs -->
+  <url type="bugtracker">https://github.com/kriptolix/Poliedros/issues</url>
+  <!-- Should link a FAQ page for this software, to answer some of the most-asked questions in detail -->
+  <!-- URLs of this type should point to a webpage where users can submit or modify translations of the upstream project -->
+  <url type="translate">https://github.com/kriptolix/Poliedros/tree/master/po</url>  
+
+  <translation type="gettext">poliedros</translation>
+  <!-- All graphical applications having a desktop file must have this tag in the MetaInfo.
+     If this is present, appstreamcli compose will pull icons, keywords and categories from the desktop file. -->
+  <launchable type="desktop-id">io.github.kriptolix.Poliedros.desktop</launchable>
+  <!-- Use the OARS website (https://hughsie.github.io/oars/generate.html) to generate these and make sure to use oars-1.1 -->
+  <content_rating type="oars-1.1" />
+
+  <!-- Applications should set a brand color in both light and dark variants like so -->
+  <branding>
+    <color type="primary" scheme_preference="light">#ff00ff</color>
+    <color type="primary" scheme_preference="dark">#993d3d</color>
+  </branding>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/kriptolix/Poliedros/blob/master/data/screenshots/dark-basic-sidebar.png</image>
```

Comment:
never reference from the tip of a branch. It's guaranteed to break.

Reply:
also this is not a URL to an image, but to the github page.

Reply:
This is the first time I've made a submission to flathub, this is my first flatpak. There are a lot of things that I'm still understanding how works, so I apologize. What would be the correct way to reference a screenshot image?